package.json

```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "react": "^16.4.1",
-    "react-dom": "^16.4.1"
+    "react-dom": "^16.4.1",
+    "react-scripts": "1.1.4"
```

Comment:
Is `react-scripts` needed? Taking a look at it's [dependencies](https://github.com/firstlookmedia/react-scripts/blob/master/package.json), it is extremely heavy. Can we make do with pulling just the few packages from there that we'll be using? Taking a look at the current code, it looks like only the base React is used.